### PR TITLE
CAN RX Implementation and Drawing improvements

### PR DESCRIPTION
**Pull request by rickyelopez**
Implements #6 

Implement CAN message reception. First step towards getting a full CAN receiver set up.
Also improves drawing and implements some of the pages that the old steering wheel had.

### CODE REVIEW

File: components/steering_wheel/include/Utility.h [hunk 1]
```diff
@@ -55,3 +66,118 @@ static inline uint16_t u32CountLeadingZeroes(uint32_t x) { uint16_t c = 0U; whil
             (bit) = false;              \
         }                               \
     } while (zero())
+
+
+// FLAGS
```
Comment (rickyelopez) [inline components/steering_wheel/include/Utility.h:71]: @JoshLafleur FYI this is the API I was referring to that you could use for the IO stuff

Comment (JoshLafleur) [reply components/steering_wheel/include/Utility.h:71]: Regarding what we spoke about for the bitfield correct? This is the workaround to not being able to reference elements of a bitfield? To be frank I am a bit confued on the conversion from name to flag, the name would it be a define? or one of the elements of the bitfield? Would like to see more the workings of this

Comment (rickyelopez) [reply components/steering_wheel/include/Utility.h:71]: Discussed offline

**Comment by rickyelopez** [comment]
@JoshLafleur want to take a look? Big PR so no worries if not, we can just merge.

**Review by JoshLafleur — APPROVED** [review]

**Comment by JoshLafleur** [comment]
@rickyelopez Want to update to current master and merge?

**Comment by rickyelopez** [comment]
Confirmed builds still pass after rebasing. Not able to confirm that it actually works since I don't have any hardware with me right now. I think we should still merge, I can fix it later if it breaks anything

**Pull request merged**